gui/src/route_printout.cpp

```diff
@@ -0,0 +1,245 @@
+/***************************************************************************
+ *   Copyright (C) 2012 by David S. Register                               *
+ *   Copyright (C) 2025 by NoCodeHummel                                    *
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ *   This program is distributed in the hope that it will be useful,       *
+ *   but WITHOUT ANY WARRANTY; without even the implied warranty of        *
+ *   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the         *
+ *   GNU General Public License for more details.                          *
+ *                                                                         *
+ *   You should have received a copy of the GNU General Public License     *
+ *   along with this program; if not, write to the                         *
+ *   Free Software Foundation, Inc.,                                       *
+ *   51 Franklin Street, Fifth Floor, Boston, MA 02110-1301,  USA.         *
+ **************************************************************************/
+
+#include <iostream>
+
+#include <wx/wxprec.h>
+
+#ifndef WX_PRECOMP
+#include <wx/wx.h>
+#endif  // precompiled headers
+#ifdef __WXMSW__
+// #include "c:\\Program Files\\visual leak detector\\include\\vld.h"
+#endif
+
+#include <wx/print.h>
+#include <wx/printdlg.h>
+#include <wx/artprov.h>
+#include <wx/stdpaths.h>
+#include <wx/intl.h>
+#include <wx/listctrl.h>
+#include <wx/aui/aui.h>
+#include <wx/dialog.h>
+#include <wx/progdlg.h>
+#include <wx/brush.h>
+#include <wx/colour.h>
+
+#include <wx/dialog.h>
+
+#include "navutil.h"
+#include "dychart.h"
+
+#ifdef __WXMSW__
+#include <stdlib.h>
```

Comment:
standard includes should be ahead of other includes: https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes

Yes, we use <> for wxwidgets includes. Live with it.